frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -147,6 +149,14 @@ class BitBox02 extends Component<Props, State> {
                     break;
             }
         });
+        import('./components/password-entry/password-entry')
+            .then(({ PasswordEntry }) => {
+                if (!this.PasswordEntry) {
```

Comment:
why this check? when can it be false?

Reply:
initially before loading the module during runtime this would be null. But outdated now anyways as we change to statically importing the module anyways